synchronization/src/main/java/de/cyface/synchronization/HttpConnection.java

```diff
@@ -18,15 +18,16 @@
  */
 package de.cyface.synchronization;
 
-import static de.cyface.persistence.Constants.DEFAULT_CHARSET;
```

Comment:
Wenn ich meinen Auto-Formatter hier drüber laufen lasse formatiert er mir es wieder zurück zum vorherigen Stand. Ich vermute du hattest versehentlich den Eclipse Formatter in dem Projekt bei dir lokal nicht korrekt eingestellt (unsere Formatter Datei hinterlegt)

Reply:
No I did not. I am switching from PC to PC so often and touching this project so little that Android Studio has not been set up correctly everywhere.

Reply:
I auto-ordered the imports back to the formatter rules, see new commit.